aries-site/src/layouts/content/WCAGRuleSummary.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable react/prop-types */
+import { Box, NameValueList, NameValuePair, Text } from 'grommet';
+import {
+  StatusCriticalSmall,
+  StatusGoodSmall,
+  StatusWarningSmall,
+  CircleInformation,
+} from 'grommet-icons';
+
+const STATUS_MAP = {
+  passed: {
+    label: 'Passed',
+    icon: <StatusGoodSmall color="status-ok" />,
```

Comment:
I think we can mark these svg's as decorative since they are placed next to text that communicates their meaning and the text will be read out by a screenreader.
We can add `alt=""` to the svgs to mark as decorative.
https://www.w3.org/WAI/tutorials/images/decorative/

Reply:
good idea added